WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEpilogueFragment.java

```diff
@@ -211,6 +211,11 @@ public void onResume() {
         }
     }
 
+    @Override
+    protected boolean isJetpackAppLogin() {
+        return mDoLoginUpdate && BuildConfig.IS_JETPACK_APP;
```

Comment:
@ashiagr - This is only a suggestion, so feel free to ignore. Should we inject ConfigWrapper instead of direct access to BuildConfig? I go back & forth when we talk about Fragments - since we generally don't unit test them.

Reply:
Right, we don't test fragments as such, so I didn't use the ConfigWrapper. I'm actually indifferent to this and can use either of the options. 

Suggestion applied in df37ee4 🙂.

Reply:
@ashiagr - thanks for the change.(Even though you didn't have to do it) :)